app/views/documents/index.html.erb

```diff
@@ -38,9 +38,9 @@
                 <%= form.submit "Save", class: "govuk-button", data: { module: "govuk-button" } %>
                 <%= link_to 'Cancel', planning_application_path(@planning_application), class: "govuk-button govuk-button--secondary" %>
               </p>
-           </div>
-      <% end%>
-    <% end %>
+        <% end%>
+     <% end %>
+    </div>
```

Comment:
I think the indentation is wrong here, and making it hard for me to see if everything is closed correctly.

Reply:
You were right: the indentation was wrong. I've force-pushed a fix for it.